Combobox/ComboPopup.js

```diff
@@ -76,7 +76,11 @@ define([
 		 * @protected
 		 */
 		cancelHandler: function () {
+			// INFO: resetting any selected items.
+			this.combobox.list.selectedItems = [];
 			this.combobox.closeDropDown();
+			// cont: then ask to validate, so widget's value and inputNode get updated as well.
+			this.combobox._validateMultiple(true);
```

Comment:
Sorry, but I don't understand what you are referring to. `cancelHandler` function runs only when user press "Cancel" button, which is visible only if `selectionMode=multiple`.
In `selectionMode=single` we don't have any button shown.